client.py

```diff
@@ -43,6 +43,21 @@ class QueryBuilder:
     VALID_SORT_ORDERS = ["asc", "desc"]
     VALID_SORT_FIELDS = ["publication_date", "indexed_date"]
 
+    class Aggregators(Enum):
+        DAILY_COUNTS = {
+            "dailycounts": {
+                "date_histogram": {
+                    "field": "publication_date",
+                    "calendar_interval": "day",
+                    "min_doc_count": 1,
+                }
+            }
+        }
+        TOP_LANGS = {"toplangs": {"terms": {"field": "language.keyword", "size": 100}}}
+        TOP_DOMAINS = {
+            "topdomains": {"terms": {"field": "canonical_domain.keyword", "size": 100}}
```

Comment:
Field changing from `canonical_domain` to `canonical_domain.keyword`?

Reply:
Yes! In trying to benchmark the latency of these operations I noticed that aggregating on the .keywords field of canonical domain made the query return significantly faster- ultimately it made the overview query almost twice as fast.